pycqed/analysis/thresholding_toolbox.py

```diff
@@ -0,0 +1,552 @@
+import numpy as np
```

Comment:
This file would benefit from a generic comment as to what is (and is not) contained in this file and what it's purpose is.